### PR TITLE
Update lockfile with preferences

### PR DESCRIPTION
**Pull request by sallner**
Should be reviewed after #27 is merged.

Should then be applied to batou.

### CODE REVIEW

**Comment by icemac** [comment]
This PR is a fix for https://github.com/flyingcircusio/batou/issues/145.

**Review by ctheune — APPROVED** [review]

**Comment by ctheune** [comment]
I'm happy so far, please rebase so that we can see Github Actions approving this, too.

File: src/appenv.py [hunk 1]
```diff
@@ -121,13 +121,59 @@ def ensure_venv(target):
...
+                if not line.startswith("# appenv-python-preference: "):
+                    continue
+                preferences = line.split(':')[1]
+                preferences = [x.strip() for x in preferences.split(',')]
+                preferences = list(filter(None, preferences))
+                break
+    if not preferences:
+        # We have no preferences defined, use the current python.
+        print("Update lockfile with with {}.".format(current_python))
+        print("If you want to use a different version, set it as via")
```
[the remark this answers is not in the document]
Comment (ctheune) [reply src/appenv.py:145]: Forget it, I can live with the wording as is, I'd rather start using it to see whether this is helpful in day-to-day work.

**Comment by icemac** [comment]
Rebased to master.

**Comment by icemac** [comment]
@ctheune CI is green, so feel free to merge this PR.

**Pull request merged**